File: WebServer/WebServer/Dto/OrderShowDto.cs
namespace WebServer.Dto
{
    public class OrderShowDto
    {
        public long Id { get; set; }
        public List<OrderArticleDto> Articles { get; set; }

        public long UserId { get; set; }

        public string Comment { get; set; }

        public string DeliveryAddress { get; set; }

        public DateTime OrderTime { get; set; }
        public DateTime DeliveryTime { get; set; }
        public bool IsDelevered { get; set; }

        public double Price { get; set; }
    }
}

File: web2projekat/src/models/OrderShowDto.js
export default class OrderShowDto {
    constructor(data) {
        this.Id = data.id;
        this.Articles = data.articles;
        this.UserId = data.userId;
        this.Comment = data.comment;
        this.DeliveryAddress = data.deliveryAddress;
        this.Price = data.price;
        this.OrderTime = data.orderTime;
        this.DeliveryTime = data.deliveryTime;
        this.IsDelevered = data.isDelevered;
    }
}